blockstack_client/rpc.py

```diff
@@ -1092,6 +1092,7 @@ def GET_name_history(self, ses, path_info, name ):
         qs_values = path_info['qs_values']
         start_block = qs_values.get('start_block', None)
         end_block = qs_values.get('end_block', None)
+        page = int(qs_values.get('page', 0))
```

Comment:
Can we require that either `start_block` and `end_block` are present, or that `page` is present, but not both?  Also, we should think about deprecating `start_block` and `end_block` in favor of pagination.